src/content/en/updates/2012/12/WebRTC-hits-Firefox-Android-and-iOS.md

```diff
@@ -14,9 +14,9 @@ book_path: /web/updates/_book.yaml
 
 <p>In particular, we're really excited to see WebRTC arriving on multiple browsers and platforms.</p>
 
-<p><code>getUserMedia</code> is available now in Chrome with no flags, as well as Opera, and Firefox Nightly/Aurora (though for Firefox you'll need to <a href="https://hacks.mozilla.org/2012/11/progress-update-on-webrtc-for-firefox-on-desktop/comment-page-1/#comment-1851192" title="Progress update on WebRTC for Firefox on desktop">set preferences</a>). Take a look at the cross-browser demo of <code>getUserMedia</code> at <a href="http://www.simpl.info/gum" title="Simple cross-platform getUserMedia demo">simpl.info/gum</a>&mdash;and check out Chris Wilson's <a href="http://webaudiodemos.appspot.com/" title="">amazing examples</a> of using <code>getUserMedia</code> as input for Web Audio.</p>
+<p><code>getUserMedia</code> is available now in Chrome with no flags, as well as Opera, and Firefox Nightly/Aurora (though for Firefox you'll need to <a href="https://hacks.mozilla.org/2012/11/progress-update-on-webrtc-for-firefox-on-desktop/comment-page-1/#comment-1851192" title="Progress update on WebRTC for Firefox on desktop">set preferences</a>). Take a look at the cross-browser demo of <code>getUserMedia</code> at <a href="https://simpl.info/gum" title="Simple cross-platform getUserMedia demo">simpl.info/gum</a>&mdash;and check out Chris Wilson's <a href="http://webaudiodemos.appspot.com/" title="">amazing examples</a> of using <code>getUserMedia</code> as input for Web Audio.</p>
 
```

Comment:
Two comments. First can you break the lines at 80 characters. Also, method names need to end with ().